app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,10 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <color name="BackgroundGreenPrimary">#4C9C4B</color>
```

Comment:
use camelCase for constants `backgroundGreenPrimary` and `greenSecondary`

Reply:
Kotlin recommended me as to set it as below:

`<color name="background_green_primary">#4C9C4B</color>`

It's ok?